package.json

```diff
@@ -37,9 +37,7 @@
     "build": "tsc",
     "postbuild": "cp src/libinchi.so.1.06.00 dist/tsc/src",
     "serve": "node dist/tsc/src/server.js",
-    "test:headers": "node dist/tsc/tests/headers.test.js",
-    "test:ffis": "node dist/tsc/tests/ffis.test.js",
-    "test": "yarn test:headers && yarn test:ffis",
+    "test": "jest tests/ffis.test.ts && jest tests/headers.test.ts",
```

Comment:
- Try to figure out why the earlier command wasn't working.
- Look into https://jestjs.io/docs/cli#--runinband also.